runtime/js/11_workers.js

```diff
@@ -9,10 +9,12 @@ const {
   ObjectPrototypeIsPrototypeOf,
   String,
   StringPrototypeStartsWith,
+  SymbolFor,
   SymbolIterator,
   SymbolToStringTag,
 } = primordials;
 import * as webidl from "ext:deno_webidl/00_webidl.js";
+import { createFilteredInspectProxy } from "ext:deno_console/01_console.js";
```

Comment:
This is likely where the problem is -- we are requiring deno_console from runtime. We can temporarily work around it by adding `deno_console` to the `url_ops` benchmark but we may need move this function into runtime.

Reply:
Anyway, I deleted the Worker's inspect support #21264